ajo/personalization-client-side/README.md

```diff
@@ -63,13 +76,50 @@ function sendDisplayEvent(decision) {
               scopeDetails: scopeDetails,
             },
           ],
+          propositionEventType: {
+            display: 1
+          },
+        },
+      },
+    },
+  });
+}
+```
+
+6. For code based experience campaigns, click interaction events must manually be sent to indicate when an element has been clicked. This is done via the `sendEvent` command.
+
+```javascript
+function sendClickEvent(text, proposition) {
+  const { id, scope, scopeDetails = {} } = proposition;
+
+  alloy("sendEvent", {
+    xdm: {
+      eventType: "decisioning.propositionInteract",
+      _experience: {
+        decisioning: {
+          propositions: [
+            {
+              id: id,
+              scope: scope,
+              scopeDetails: scopeDetails,
+            },
+          ],
+          propositionEventType: {
+            interact: 1
+          },
+          propositionAction: {
+            label: text
+          },
         },
       },
     },
   });
 }
 ```
 
+## Personalization Payloads
+Please refer to the sample personalization payloads in the [Personalization Payloads](../PersonalizationPayloads.md) file
```

Comment:
What is the point of this section?  Why would the user want to refer to the page mentioned?

I think adding a bit more context here is important. Also, not sure it warrants an entire heading does it?

Reply:
I'd move this reference into the Key Observations section under the Flow Diagram, where you can say e.g. "Please refer to [Sample Adobe Experience Platform Edge Network Personalization Payloads] for examples of the Edge Network API request and responses in steps 3, 4 and 5 of the flow".

Reply:
I have moved the references under flow diagram as this, 
Please refer to [Sample Adobe Experience Platform Edge Network Personalization Payloads] for examples of the Edge Network API request and responses in the steps `Alloy->>API` and `API->>Alloy` of the flow.